Packs/Base/TestPlaybooks/playbook-Indicators-reputation-.json-Test.yml

```diff
@@ -1017,6 +1017,14 @@ tasks:
           right:
             value:
               simple: http://www.foo.bar/taz?())
+      - - operator: notContainsGeneral
+          left:
+            value:
+              simple: ${URL.Data}
+            iscontext: true
+          right:
+            value:
+              simple: https://Test.com/this'that
```

Comment:
It's not the same as the input, is that on purpose?

Reply:
Yes, when I upload the updated playbook and execute it, I get:

Before the changes (URL doesn't support apostrophes):
![image](https://user-images.githubusercontent.com/38375556/117953486-bad76080-b31e-11eb-8b01-96636dba96df.png)

After the changes (URL does support apostrophes):
![image](https://user-images.githubusercontent.com/38375556/117952072-7ac3ae00-b31d-11eb-929c-13f67b0629b1.png)